sphere-java-client-core/src/main/java/io/sphere/sdk/client/SphereClientFactory.java

```diff
@@ -95,6 +93,44 @@ public SphereClient createClient(final String projectKey, final String clientId,
     }
 
     /**
+     * Creates a client which relies on an access token and does not refresh it, it reuses an existing http client.
+     *
+     * A possible use case is keeping the clientSecret secret and/or instantiate a client which is used for few requests for a limited time.
+     * The underlying http client does not get closed by closing the client
```

Comment:
I'm certain there's a good reason behind it I'm not thinking of right now, but why?